bin/worker.js

```diff
@@ -314,6 +304,6 @@ async function main () {
 
 main().catch((err) => {
   // Top level uncaught fatal errors!
-  console.error(err);
+  console.error(err.stack);
```

Comment:
`tc-lib-monitor` does something similar on uncaught exceptions https://github.com/taskcluster/taskcluster-lib-monitor/blob/master/src/monitor.js#L23 but also reports it to Sentry. I think we can just keep that and we won't need this `.catch(...)`. I think this might end up silencing the error in Sentry.